src/utils/boolean-utils.ts

```diff
@@ -23,3 +24,11 @@ export function isUuid(value: string): boolean {
   const uuidRegex = /^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$/;
   return uuidRegex.test(value);
 }
+
+export function isFieldRequired(value: FormField): boolean {
+  if (typeof value.required !== 'object' ){
+    return  isTrue(value.required);
+  } else if ( typeof value.required === 'object' && value.required.type === 'conditionalRequired' ){
+    return true;
```

Comment:
You have formatting issues in the second "if" clause. 

Two questions:
1. What happens if the underlying condition (the condition that determines the mandatory behaviour) is not met or resolves to false?
2. What happens if the second "if" resolves to false (I know it's less likely but we should return false or better just return the expression)?